Makefile

```diff
@@ -4,24 +4,33 @@ PACKAGES_INSTALL=$(PACKAGES)
 
 SOURCE_DIRS=/util /unc /stats /cls /rgr /uns
```

Comment:
Can you please also remove this line? It is a carry over of when I copied the `Makefile` from `oml`. So my bad.